### ISSUE TITLE
Better/New Textures (Vanilla Friendly, Create like)

### ISSUE THREAD

**Issue by Fundryi** [PrinceOfAmber/Cyclic#2314]
Hi, yes, I know this is probably a very very low priority even if at all.
But I feel like the texture could need some more love, so it fits more the Vanilla style, or even similar to create.

I know you guys create texture in paint and it's not that important, but I just wanted to just suggest it x) 
I bet a lot of people would like to see this.

That is in no way meant as a critic or saying the current are bad xD please whoever made them, don't take it personally :(

**Comment by Lothrazar** [comment]
Yeah i would love some! if you make any that look nicer than what are in the let me know i will probably accept the pull request

**Comment by Lothrazar** [comment]
back in 2021 i made a list of every asset that i didnt like the look of that i wanted to get someone to refresh/redo the texture, its right here .  Anything on this list is past do for a re-draw .  or if you know any artists, the list is up here https://github.com/Lothrazar/Cyclic/projects/5

**Comment by Lothrazar** [comment]
if i find or make something good, or if anyone submits some nice stuff, then using the magic of git i can deploy new textures to all the different Minecraft versions

**Comment by The4codeblocks** [comment]
> back in 2021 i made a list of every asset that i didnt like the look of that i wanted to get someone to refresh/redo the texture, its right here . Anything on this list is past do for a re-draw . or if you know any artists, the list is up here https://github.com/Lothrazar/Cyclic/projects/5

where'd the list go
